src/api.js

```diff
@@ -123,3 +124,25 @@ export async function getSingleMovieReview(
     return reviewArray;
   }
 }
+
+// data = array of screenings, today = todays time and date in ISO format
+export function filterOutOldScreenings(data, today){
+ 
+  // run through the array and return objects containing time >= todaysTime
+  const res = data.filter(comming);
+    
+  // currentValue is reqired parameter but not needed, index is index of current object
+      function comming(currentValue, index){
+        const screening = new Date(data[index].attributes.start_time);
+          if(screening >= today ){
+            return data[index];
```

Comment:
You should be returning `true` here. This also works, because `data[index]` is truthy, but the object itself will not be used.